components/lib/VHeader.vue

```diff
@@ -141,6 +141,13 @@ header {
   text-underline-offset: 0.25rem;
 }
 
+.cta-button {
+  height: 3rem;
+  width: 14rem;
+  padding: 0;
```

Comment:
What is the zero padding doing here?

Reply:
By default our button component has this padding `padding: 1.125rem 2rem;`, we're overriding that default padding since we don't need it here